### PR TITLE
feature/basic  UI

### PR DESCRIPTION
**Pull request by blessedjasonmwanza**
## feature/basic UI
In ux console APP update, complete app functionality by implementing the following functionality

-  List all books.
-  List all people.
-  Create persons.
-  Create a book.
-  Create a rental.
-  List all rentals for a given person id.

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+
+    case option
+    when '1'
+      create_student
+    when '2'
+      create_teacher
+    else
+      puts 'Invalid input. Kindly type 1 or 2'
+    end
+  end
```
Comment (MahmoudBakr23) [inline app.rb:84]: - [x] When I try to create a new person, it threw an error after I entered the parent permission! Please double-check the `person.rb` file!
Git-Bksh__c_Users_mbakr_OneDrive_Desktop_assesss_school-library-ruby 3_18_2022 12_51_54 AM (1)

File: README.md [hunk 2]
```diff
@@ -17,7 +17,7 @@ To get a local copy up and running follow these simple example steps.
 
 - Open terminal in cloned repo directory
 
-- then run ```bundle install```.
+- then run ```ruby main.rb```.
```
Comment (MahmoudBakr23) [inline README.md:20]: - [x] In the README file, please fix the MIT link in order to maintain a professional repo. 🙏

**Review by MahmoudBakr23 — CHANGES_REQUESTED** [review]
Hello @blessedjasonmwanza 😃, 

Great work and effort you put into this project so far.

## Highlights 🚀 

There are some excellent points that you have made perfectly. Keep up! 💪
- Linters are correct
- Professional PR headline and summary
- Great UI console

However, there is always room for improvement.

> Keep in mind that your long hours of work are appreciated. However, the code review process is for improving the code you wrote, and not for criticizing it. 😄

 ## Required Changes ♻️
_Please check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

N/A

Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 2 above)
Comment (blessedjasonmwanza) [reply README.md:20]: Actioned.

File: app.rb (hunk 1 above)
Comment (blessedjasonmwanza) [reply app.rb:84]: Thanks a lot for this point out @MahmoudBakr23 . I have fixed the issue and now you can create a persons account

File: README.md (hunk 2 above)
Comment (rloterh) [reply README.md:20]: This issue is well resolved! ✔️

File: app.rb (hunk 1 above)
Comment (rloterh) [reply app.rb:84]: This issue is well resolved! ✔️

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Blessed, 

Your project is complete! Great job for making the necessary changes recommended in the previous review. There is nothing else to say other than... it's time to merge it 🍾 :shipit:. Congratulations! 🎉🥂

<br>
Cheers and Happy coding!👏 💻 🍷 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 1 more limited review per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**